app/assets/javascripts/leaflet.map.js

```diff
@@ -9,6 +13,35 @@ L.extend(L.LatLngBounds.prototype, {
   }
 });
 
+if (OSM.MAPTILER_KEY) {
+  maplibregl.setRTLTextPlugin("https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-rtl-text/v0.2.3/mapbox-gl-rtl-text.js", true);
```

Comment:
I am pretty sure this need to be changed to be served from osm.org or probably bundled like the rest...

Reply:
To answer the question from that comment, you modify `app/assets/config/manifest.js` and add:

```js
//= link @mapbox/mapbox-gl-rtl-text/mapbox-gl-rtl-text.js
```

then add to `app/assets/javascripts/osm.js.erb` something like:

```js
RTL_TEXT_PLUGIN: <%= javascript_path("@mapbox/mapbox-gl-rtl-text/mapbox-gl-rtl-text.js").to_json %>,
```

and then you can use `OSM.RTL_TEXT_PLUGIN` here in place of the mapbox URL string.